source/templates.tex

```diff
@@ -475,7 +475,7 @@
 
 template<C1 T> struct s1;              // associates \tcode{C1<T>}
 template<C1... T> struct s2;           // associates \tcode{(C1<T> \&\& ...)}
-template<C2... T> struct s3;           // associates \tcode{(C2<T> \&\& ...)}
+template<C2... T> struct s3;           // associates \tcode{C2<T...>}
```

Comment:
I don't believe this change is correct.  I can see how the algorithm in [temp] p6 yields the original outcome, but not how it would yield your suggested outcome.

Reply:
Hmm, sorry - this behaviour must have changed since the TS and I didn't notice.
Apologies.
I'll remove this change from this P/R.

Reply:
I think I dimly remember we made a change in this area.

Reply:
Now removed.